### PR TITLE
Match animCmd_GetTiles and animCmd_6

### PR DESCRIPTION
**Pull request by JaceCear**

### CODE REVIEW

File: include/global.h [hunk 1]
```diff
@@ -101,6 +101,12 @@
 // with for the calculation
 #define GBA_FRAMES_PER_SECOND 60
 
+#define SWAP_AND_NEGATE(a, b)                                                           \
```
Comment (freshollie) [inline include/global.h:104]: @jiangzhengwenjz have you seen this anywhere in katam? Is there some one liner we can use here instead?

File: src/animation_commands_extra.c [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+extern u8 gUnknown_03002A84;
+extern struct BgHeader *gUnknown_030027A0[];
+
+// (-1)
+s32 animCmd_GetTiles(void *cursor, Sprite *sprite)
+{
+    ACmd_GetTiles *cmd = (ACmd_GetTiles *)cursor;
+    sprite->unk14 += AnimCommandSizeInWords(ACmd_GetTiles);
+
+    if ((sprite->unk10 & 0x80000) == 0) {
```
Comment (freshollie) [inline src/animation_commands_extra.c:20]: Can write these as `!(var &  0x80000)`

File: src/animation_commands_extra.c [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            sprite->unk0
+                = (u32)&gUnknown_03002794->tiles_8bpp[cmd->tileIndex * TILE_SIZE_8BPP];
+            sprite->unk8 = cmd->numTilesToCopy * TILE_SIZE_8BPP;
+        } else {
+            sprite->unk0
+                = (u32)&gUnknown_03002794->graphics[cmd->tileIndex * TILE_SIZE_4BPP];
+            sprite->unk8 = cmd->numTilesToCopy * TILE_SIZE_4BPP;
+        }
+
+        gUnknown_030027A0[gUnknown_03002A84] = (struct BgHeader *)sprite;
```
Comment (freshollie) [inline src/animation_commands_extra.c:32]: Huh, interesting, so this is more like sprite/bg header? If you can think of a better name for this, go for it

**Review by freshollie — APPROVED** [review]

File: include/global.h (hunk 1 above)
Comment (jiangzhengwenjz) [reply include/global.h:104]: > @jiangzhengwenjz have you seen this anywhere in katam? Is there some one liner we can use here instead?

In katam it's juts swap. No negate.

Comment (freshollie) [reply include/global.h:104]: So what @JaceCear has written here can’t be simplified? I wondered if there was some weird trick in agbcc

Comment (jiangzhengwenjz) [reply include/global.h:104]: > So what @JaceCear has written here can’t be simplified? I wondered if there was some weird trick in agbcc

https://github.com/jiangzhengwenjz/katam/blob/master/include/global.h#L53 This is what I have in katam. So it's really different.

**Pull request merged**